src/components/SharingTags.js

```diff
@@ -0,0 +1,54 @@
+import startCase from 'lodash.startcase'
+import PropTypes from 'prop-types'
+import React from 'react'
+import Helmet from 'react-helmet'
+import { connect } from 'react-redux'
+
+import offenses from '../util/offenses'
+import { getAgency } from '../util/ori'
+import { getPlaceInfo } from '../util/place'
+import usa from '../util/usa'
+
+const SharingTags = ({ agency, crime, place, since, until }) => {
```

Comment:
I believe so. It looks like helmet is a sort of global object and so this plays nicely and doesn't change the current title tags.

To add onto the points above about why I like this approach, for the sharing component in particular, it gives us the ability to wrap it with `react-redux`'s connect and therefore pull out any information we need across the entire site without needing to keep track of what we're passing in